docs/testplan/Smartswitch-test-plan.md

```diff
@@ -404,9 +431,9 @@ On Switch: Showing example of one DPU pcie link
 
 root@sonic:/home/cisco# show platform pcieinfo -c
 
-root@sonic:/home/cisco# echo 1 > /sys/bus/pci/devices/0000:1a:00.0/remove
+root@sonic:/home/cisco# config chassis modules shutdown DPU<DPU_NUM>
```

Comment:
This can be considered as both, since while shutting down it detach the pcie.
While powering up it does rescan. That is why we dont need to have dpu power cycle case separately.